openpype/settings/entities/schemas/projects_schema/schemas/schema_nuke_publish.json

```diff
@@ -371,6 +371,151 @@
 
             ]
         },
+        {
+            "type": "label",
+            "label": "^ Settings and for <span style=\"color:#FF0000\";><b>ExtractReviewDataMov</b></span> is deprecated and will be soon removed.  <br> Please use <b>ExtractReviewDataBakingStreams</b> instead."
```

Comment:
Should we mention when **soon** is? 

Can we e.g. mention a version number when it should be removed? And I've seen "deprecated by version x.y.z" messages here and there - do we have a good "todo" tracking list for those removals so we actually remove those when intended? @iLLiCiTiT do you know? If not, any ideas on how we could best set that up?

Reply:
I would suggest avoiding the restriction for now since we don't currently have a proper workflow for tracking TODOs. This theme is quite complex. I recommend opening a discussion topic in the community as it has the potential to be very lively.

Reply:
Should we still name the version we're expecting to deprecate it like we've done for other deprecations recently?